GameData/ROCapsules/PartConfigs/Apollo D2/D2ServiceModule.cfg

```diff
@@ -0,0 +1,139 @@
+PART
+{
+	name = ROC-D2ServiceModule
+	module = Part
+	author = Mcdouble, Vader111
+
+	RSSROConfig = True
+
+	MODEL
+	{
+		model = ROCapsules/Assets/McDouble/D2/D2ServiceModule
+	}
+
+	scale = 1
+	rescaleFactor = 1.0
+
+	node_stack_top = 0.0, 1.375, 0.0, 0.0, 1.0, 0.0, 2
+	node_stack_bottom = 0.0, -1.375, 0.0, 0.0, -1.0, 0.0, 2
+
+	CoMOffset = 0.0, -1.4, 0.0
+
+	attachRules = 1,0,1,1,0
+
+	//	============================================================================
+	//	Title, Description, Category, Techs
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = FuelTank
+	subcategory = 0
+
+	title = D-2 Service Module
+	manufacturer = General Electric
+	description = Service Module for D-2 vehicle, carries fuel and other vital systems.
+	tags = D2, apollo, GE
+
+	bulkheadProfiles = size2
+
+	//	============================================================================
+	//	Thermo, Durability
+	//	============================================================================
+
+	mass = 1.2
+	dragModelType = default
+	maximum_drag = 0.20
+	minimum_drag = 0.15
+	angularDrag = 2
+	crashTolerance = 10
+	breakingForce = 250
+	breakingTorque = 250
+	maxTemp = 673.15
+	skinMaxTemp = 773.15
+	fuelCrossFeed = true
+	emissiveConstant = 0.6 // not too absorptive for reentry
+	thermalMassModifier = 1.0
+	skinMassPerArea = 4.0
+	radiatorHeadroom = 0.5
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 10000.0
+		basemass = -1
+		TANK
+		{
+			name = ElectricCharge
+			amount = 2000
+			maxAmount = 2000
+		}
+		TANK
+		{
+			name = MMH
+			amount = 234.5
+			maxAmount = 234.5
+		}
+		TANK
+		{
+			name = NTO
+			amount = 280.5
+			maxAmount = 280.5
+		}
+	}
+}
+
+@PART[ROC-D2ServiceModule]:AFTER[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	MODULE:NEEDS[TacLifeSupport]
+	{
+		name = TacGenericConverter
+		converterName = Fuel Cell 1
+		StartActionName = Start Fuel Cell 1
+		StopActionName = Stop Fuel Cell 1
+		conversionRate = 4.0
```

Comment:
This is very inconsistent with the rate in the Kerbalism config. We basically don't use TACLS anymore so it's not a big deal (and I changed what was submitted in the ROKerbalism config because a straight clone from Gemini won't work--this is a lunar-orbital 3 person craft, at a higher tech level) but I'm just pointing this out. <no action necessary here>